src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -1273,6 +1275,19 @@ private bool IsQueryMatch(string query, string symbolName)
                 new Protocol.LanguageServer.RunspaceDetails(e.NewRunspace));
         }
 
+        /// <summary>
+        /// Event hook on the PowerShell context to listen for changes in script execution status
+        /// </summary>
+        /// <param name="sender">the PowerShell context sending the execution event</param>
+        /// <param name="e">details of the execution status change</param>
+        /// <returns></returns>
```

Comment:
nit: maybe delete this line unless others in the file have the same

Reply:
Looking through the file and the codebase and the documentation, I noticed we didn't have as much documentation as in the PowerShell codebase (in fact, the web docs on the code base are out of date and refer to classes that no longer exist). I was thinking that anything I contribute I should document. Ideally I'd want to see comments documenting most things, just so we can make contributing easier, but for now I think some documentation is better than none.

Reply:
@rjmholt Definitely agree! Documentation is an area needs a lot of work here :)

In this case, I believe he's referring to just the empty `returns` tag.  I'd also prefer if they were dropped to keep it consistent. If I remember correctly even if they were public API's, the rule for returns documentation doesn't hit on properties or void methods.

Reply:
Ah I see. Yes agreed!